src/translations/es.json

```
@@ -10,192 +10,191 @@
+      "waitingForGameToStart": "Esperando que comience la partida",
+      "leaveGame": "Abandonar la partida",
+      "opponentMayCounter": "El rival puede contrarrestar",
+      "opponentMustResolve": "El rival debe resolver",
+      "opponentIsDiscarding": "El rival se está descartando",
+      "opponentChoosingFromScrap": "El rival está eligiendo del mazo de descarte",
+      "opponentPlayingFromDeck": "El rival está jugando desde el mazo",
+      "opponentMustDiscardJack": "El rival debe descartar la Sota",
+      "opponentConsideringStalemate": "El rival está considerando la solicitud de empate",
       "gameUnavailable": {
-        "title": "Oops!",
-        "content": "Looks like this game isn't available."
+        "title": "Ups!",
+        "content": "Parece que esta partida no está disponible."
       }
     },
     "dialogs": {
       "reauthentication": {
-        "reconnectToGame": "Vuelve a conectarte al juego",
-        "youHaveDisconnected": "Te has desconectado por inactividad. Inicie sesión nuevamente para reanudar su sesión",
+        "reconnectToGame": "Volver a conectarse a la partida",
+        "youHaveDisconnected": "Te has desconectado por inactividad. Inicia sesión de nuevo para reanudar.",
         "username": "Nombre de usuario",
         "password": "Contraseña",
-        "leaveGame": "Dejó el juego",
+        "leaveGame": "Abandonar la partida",
         "login": "Iniciar sesión"
       },
       "scrapDialog": {
         "close": "Cerrar",
-        "scrapPile": "Pila de chatarra",
-        "noCards": "No hay cartas en la pila de chatarra."
+        "scrapPile": "Mazo de descarte",
+        "noCards": "No hay cartas en el mazo de descarte."
       },
       "cardListSortable": {
         "sort": {
-          "label": "Clasificar",
+          "label": "Ordenar",
           "chronologically": "Cronológicamente",
           "rank": "Por Rango"
         }
       },
       "four": {
         "discardTwoCards": "Descartar dos cartas",
-        "discard": "Desechar",
-        "oppponentHasResolved": "Tu oponente ha resuelto un Four One-Off. Debes descartar dos cartas. Haz clic para seleccionar cartas para descartar."
+        "discard": "Descartar",
+        "oppponentHasResolved": "El rival ha usado la habilidad Única del Cuatro. Debes descartar dos cartas. Haz clic para seleccionar las cartas para descartar."
       },
       "counterDialogs": {
         "counterTitle": "Oportunidad de contrarrestar",
-        "chooseTwoTitle": "Escoge dos",
-        "cannotCounterTitle": "No Puede contrarrestar",
-        "selectCard": "¿Con cuáles dos te gustaría contrarrestar? (Haz clic en la tarjeta)",
-        "opponentPlayed": "Tu oponente ha jugado el",
-        "oneOff": "como un one-off",
-        "target": "apuntar ",
+        "chooseTwoTitle": "Elige dos",
+        "cannotCounterTitle": "No se puede contrarrestar",
+        "selectCard" : "¿Con qué Dos te gustaría contrarrestar? (Haz clic en la carta)",
```

But no whitespace before the :
```suggestion
        "selectCard": "¿Con qué Dos te gustaría contrarrestar? (Haz clic en la carta)",
```